<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SDM-Aleph</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Service</string>
   </property>
   <widget class="QLineEdit" name="serviceLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="checkServiceButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>21</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Task</string>
   </property>
   <widget class="QComboBox" name="classValComboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Target attribute value:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rankedRadioButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ranked data</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="labelledRadioButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Labelled data</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>321</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Search constraints</string>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Min subgroup size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Max number of terms</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="minSizeSGSpinBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxNumTermsSpinBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>48</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Cutoff</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="cutoffSpinBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>300</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
